wenet/squeezeformer/convolution.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) 2020 Mobvoi Inc. (authors: Binbin Zhang, Di Wu)
```

Comment:
please note you should change the `company` and `author`.

Reply:
Or append

Reply:
Copyrights is added.